### ISSUE TITLE
Nvda does not properly advertise the software version box balabolka

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9302]
### Steps to reproduce:
Open the software balabolka
In the Help menu open the option about
### Actual behavior:
The nvda does not read the dialog automatically.
Press nvda + b.
The nvda reading the window but not reading the entire window cuts the beginning of the text where there is the software version number.
If you use text navigation you can read the entire window, including the first part where the software is.
### Expected behavior:
Nvda should read the window about automatically when opening, just as pressing nvda + b should read the entire window without cutting a trexo.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
17134
#### Name and version of other software in use when reproducing the issue:
balabolka 2.15.0.687
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
log:
[about_balabolka.txt](https://github.com/nvaccess/nvda/files/2885957/about_balabolka.txt)

**Comment by gregjozk** [comment]
Hi,

I contacted a developer of Balabolka about accessibility issues, which are in Balabolka and He said, that he does not have experiences with programming with accessibilitiy in mind and does not know how to resolve these issues. unfortunately I don't have enough skills to help him how to implement accessibility APIs and other things which should be corrected within Balabolka. I think, that NVDA cannot read controls and other things in Balabolka because they are incorectly labled etc. But I don't want to critisize a developer of Balabolka, cause all in all it is a very useful program.

**Comment by Adriani90** [comment]
@fernando-jose-silva how is this now with NVDA 2020.1 and last version of this software?
Please make sure that "report object descriptions" is enabled in NVDA's object presentation settings.

**Comment by fernando-jose-silva** [comment]
nvda did not automatically read the dialog window, however it seemed to read the entire window by pressing nvda + b.

**Comment by Adriani90** [comment]
Ok Fernando thank you for reporting. It seems the specific issue here is solved now. Regarding automatically read a dialog when it opens, there is another issue already in place. I am closing this now as works for me.
